docs/workload-development-kit/workload-environment.md

```diff
@@ -50,6 +48,33 @@ During the development cycle, testing a workload on a nonproduction tenant can b
 
 :::image type="content" source="./media/workload-environment/cloud-mode-diagram.png" alt-text="Diagram of the cloud mode architecture." lightbox="./media/workload-environment/cloud-mode-diagram.png":::
 
+### Workload NuGet package structure
+
+The workload is packaged as a NuGet package, combining backend and frontend components. The structure adheres to specific naming conventions and is enforced by Fabric for consistency across upload scenarios.
+The NuGet package designed to represent workloads is structured to include both backend and frontend components.
+
+#### Backend Structure
+
+The backend segment comprises XML files that define the Workload and its associated Items, which are essential for registration within the Fabric system.
+
+##### Key Components:
+- `WorkloadManifest.xml` - The workload configuration file, required to have this exact name for Fabric's verification.
+- `Item1.xml`, `Item2.xml`, `...` - Manifests for individual items with flexible naming, following the XML format.
```

Comment:
Suggest using the same phrasing as the 'WorkloadManifest.xml' in line above (61) , "following the XML format" is a bit unclear in my opinion.